libs/sdk-ui-kit/src/Form/InputPure.tsx

```diff
@@ -0,0 +1,205 @@
+// (C) 2020 GoodData Corporation
+import React from "react";
+import cx from "classnames";
+import NativeListener from "react-native-listener";
+import bindAll from "lodash/bindAll";
+import noop from "lodash/noop";
+
+import { ENUM_KEY_CODE } from "./typings";
+
+/**
+ * @internal
+ */
+
+export interface InputPureProps {
+    autofocus: boolean;
+    className: string;
+    clearOnEsc: boolean;
+    disabled: boolean;
+    hasError: boolean;
+    hasWarning: boolean;
+    isSearch: boolean;
+    isSmall: boolean;
+    maxlength: number;
+    onChange: (value: string | number, e?: React.ChangeEvent<HTMLInputElement>) => void;
+    onEscKeyPress: () => void;
+    onEnterKeyPress: () => void;
+    onBlur: () => void;
+    onFocus: () => void;
+    placeholder: string;
+    prefix: string;
+    readonly: boolean;
+    suffix: string;
+    label: string;
+    labelPositionTop: boolean;
+    value: string | number;
+}
+/**
+ * @internal
+ */
+export class InputPure extends React.PureComponent<InputPureProps> {
+    public inputNodeRef: HTMLInputElement;
+
+    static defaultProps = {
+        autofocus: false,
+        className: "",
+        clearOnEsc: false,
+        disabled: false,
+        hasError: false,
+        hasWarning: false,
+        isSearch: false,
+        isSmall: false,
+        maxlength: 255,
+        onChange: noop,
+        onEscKeyPress: noop,
+        onEnterKeyPress: noop,
+        onBlur: noop,
+        onFocus: noop,
+        placeholder: "",
+        prefix: "",
+        readonly: false,
+        suffix: "",
+        label: "",
+        labelPositionTop: false,
+        value: "",
+    };
+
+    constructor(props: InputPureProps) {
+        super(props);
+
+        bindAll(this, ["onClear", "onChange", "onKeyPress"]);
```

Comment:
Please use arrow functions instead of `bindAll`.

Reply:
fixed